protocols/datagram.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+// TODO:
+// Come up with a better name for this.
+package beavertail;
+
+message DatagramPush {
+  string busID                    = 10;
```

Comment:
I think the field numbers should all be divided by ten - here's an excerpt from the protobuf documentation

> Note that field numbers in the range 1 through 15 take one byte to encode, including the field number and the field's type (you can find out more about this in Protocol Buffer Encoding). Field numbers in the range 16 through 2047 take two bytes. So you should reserve the numbers 1 through 15 for very frequently occurring message elements. Remember to leave some room for frequently occurring elements that might be added in the future.

Reply:
Ah didn't see that. Def think this is a good idea.